index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * addOne returns input + 1
+ * @param {number} num
+ * @returns {number}
+ */
+const addOne = (num) => num + 1
+
+/**
+ * @param {string} str string to be left-padded
+ * @param {number} len length of target string to be returned
+ * @param {character} cha character or string to pad on left
+ * @returns {string}
+ */
+const leftPad = (str, len, cha = ' ') => {
+  if (str.length > len) throw new Error('cannot pad, argument is too long')
+  if (cha.length > 1)
```

Comment:
Please use curly braces and semicolons or configure Prettier to insert them for you. Makes the code more readable, maintainable, and predictable.